.gitignore

```diff
@@ -14,3 +14,6 @@ test-results.xml
 test_config.json
 .idea
 testUserData
+
+src/browser/*.png
+src/browser/*.svg
```

Comment:
can you explain about this two lines? are images being generated within the src folder or something like that?

Reply:
Yeah, it's using the webpack image loader, just like it generates the `*_.bundle.js` files. So I have to ignore them since they're generated.